devicetypes/plaidsystems/spruce-sensor.src/spruce-sensor.groovy

```diff
@@ -293,6 +302,11 @@ def setConfig(){
     sendEvent(name: 'configuration',value: configInterval, descriptionText: "Configuration initialized")
 }
 
+def installed(){
+	//check every 1 hour + 2mins
+    sendEvent(name: "checkInterval", value: 1 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
```

Comment:
So how often does this device check-in (generate an event) when idle? Currently if our platform doesn't hear from this device in 62min (your checkInterval value) it'll mark it `OFFLINE`